code/game/objects/items/storage/pouch.dm

```diff
@@ -505,7 +505,7 @@
 	icon_state = "medkit_som"
 	sprite_slots = null
 
-/obj/item/storage/pouch/medical_injectors
+/obj/item/storage/pouch/medical_injectors/firstaid
```

Comment:
I don't get why you changed the base type path, this is just the default injector pouch that all others inherit from. I think it's fine if you just leave it as is, /firstaid is defined below anyway.